app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const app = express();
+const port = 3000
+
+//ENGINE// 
```

Comment:
Para que nuestras imagenes se puedan ver es necesario settear el public directory
```suggestion
const path = require('path');
//ENGINE// 
app.use('/public', express.static(path.join(__dirname, 'public')));
```